mico-core/src/main/java/io/github/ust/mico/core/MicoCoreApplication.java

```diff
@@ -55,4 +78,30 @@ public static void main(String[] args) {
     public RestTemplate restTemplate(RestTemplateBuilder builder) {
         return builder.build();
     }
+
+    public void onApplicationEvent(final ApplicationReadyEvent event) {
+        try {
+            MicoService kafkaFaasConnector = gitHubCrawler.crawlGitHubRepoLatestRelease(kafkaConfig.getKafkaFaasConnectorUrl());
+            List<MicoService> micoServices = micoServiceBroker.getAllVersionsOfServiceFromDatabase(kafkaFaasConnector.getShortName());
+            Optional<MicoVersion> highestKafkaFaasConnectorVersion = micoServices.stream().map(micoService -> {
+                try {
+                    return micoService.getMicoVersion();
+                } catch (VersionNotSupportedException e) {
+                    e.printStackTrace();
+                }
+                return null;
+            }).max(MicoVersion::compareTo);
+            if (kafkaFaasConnector.getMicoVersion().greaterThan(highestKafkaFaasConnectorVersion.get())) {
+                micoServiceBroker.persistService(kafkaFaasConnector);
+            }
+        } catch (IOException | VersionNotSupportedException e) {
+            e.printStackTrace();
+
+        }
+        // should not happen
+        catch (MicoServiceAlreadyExistsException e) {
+            log.info(e.getMessage());
```

Comment:
Make the logging message more specific. This cannot be distinguished from many other cases where a service already exists.